benchmarks/src/main/scala/benchmarks/Main.scala

```diff
@@ -0,0 +1,30 @@
+package benchmarks
+
+import java.lang.System.exit
+
+object Main {
+
+  val nsPerBenchmark = 3e9.toLong
+
+  def main(args: Array[String]): Unit = {
```

Comment:
I'd propose to simply run this main class with `perf`.
Eg, like this `perf -r 1000 -d stat ./Main`.